### PR TITLE
Make webhooks able to subscribe to multiple events at once

### LINKED ISSUE #502: webhooks: add `events` array field
We'd like folks to be able to sign up for more than one webhook event in the same hook:
```
{
  events: ['stream.started', 'stream.detection']
}
```
For backwards compatibility, `event` should still work when creating a webhook, but we should also update any pertinent documentation to reference `events` instead of `event`.

### PR DESCRIPTION
**Pull request by victorges**
**What does this pull request do? Explain your changes. (required)**
This is to change the webhooks spec a bit by allowing them to subscribe to multiple
events at once with an `events` array instead of a single `event` subs.

We still want to keep retrocompatibility in the API since some documents have already 
been distributed where the `event` field is used instead of `events`, but we can
actually break the backward compatibility in the database for now, since the only
webhooks that exist are ones created by Livepeer employeed for testing. 

So we'll just deploy this change and then truncate the existing prod and staging
tables so we don't have webhooks in the legacy format saved anywhere (even though
they wouldn't cause much trouble anyway, would just never be used).

For simplicity, we also don't have any index in the `events` field, but it is possible to
be made in the future with [a GIN index](https://www.postgresql.org/docs/9.4/datatype-json.html) on the specific array field, which can be used
for this `@>` operator. It didn't "just work" when I experimented with it, so I kept the simple
version with no index for now.

## -

- **How did you test each of these updates (required)**
`yarn test`

**Does this pull request close any open issues?**
Closes https://github.com/livepeer/livepeer-com/issues/502

**Checklist:**

- [x] I have read the **CONTRIBUTING** document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: packages/api/src/schema/schema.yaml [hunk 1]
```diff
@@ -58,12 +58,20 @@ components:
...
           example: 1587667174725
         event:
-          type: string
-          enum:
-            - stream.started
-            - stream.detection
-            - stream.idle
-            - recording.ready
+          writeOnly: true
+          deprecated: true
```
Comment (iameli) [inline packages/api/src/schema/schema.yaml:62]: Nice.

**Review by iameli — APPROVED** [review]
LGTM!

**Comment by adamsoffer** [comment]
Nice!

**Comment by victorges** [comment]
FTR, after deploying we need to erase the webhook table from postgres. Some step by step for safety:
 - Grab postgres IP and password from vault `LP_POSTGRES_URL`
 - Connect to postgres with`psql -h $POSTGRES_IP -U postgres`, input pwd from vault
 - Verify only internal users have webhooks created in the DB with: `SELECT DISTINCT users.data->>'email' FROM Users JOIN Webhook hook ON hook.data->>'userId' = users.id;`
 - If that's confirmed actually delete/truncate/erase the table with: `DELETE FROM Webhook;` (scary)
 - Profit

**Pull request merged**